src/utils/staking.js

```diff
@@ -203,7 +203,7 @@ export class Staking {
             validators = await this.getValidators()
         }
 
-        let totalUnstaked = new BN(balance.available, 10).sub(new BN(parseNearAmount('1'), 10))
+        let totalUnstaked = new BN(balance.available, 10).sub(new BN(parseNearAmount('2'), 10))
```

Comment:
@Patrick1904 let's use `MIN_BALANCE_FOR_GAS` from `wallet.js`

Reply:
nit: note that you generally don't need to specify `10` as a base for `new BN()`